src/NUnitFramework/framework/Attributes/ValueSourceAttribute.cs

```diff
@@ -46,7 +46,7 @@ public class ValueSourceAttribute : DataAttribute, Interfaces.IParameterDataSour
         /// <param name="sourceName">The name of the data source to be used</param>
         public ValueSourceAttribute(string sourceName)
         {
-            this.sourceName = sourceName;
+            SourceName = sourceName;
```

Comment:
Same point about TestCaseSource about including the restrictions in the XmlDocs.